runtime/leaderelection/leaderelection.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package leaderelection
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	flagEnable          = "enable-leader-election"
+	flagReleaseOnCancel = "leader-election-release-on-cancel"
+	flagLeaseDuration   = "leader-election-lease-duration"
+	flagRenewDeadline   = "leader-election-renew-deadline"
+	flagRetryPeriod     = "leader-election-retry-period"
+)
+
+// Options contains the configuration options for the leader election.
+type Options struct {
+	// Enable determines whether or not to use leader election when
+	// starting the manager.
+	Enable bool
+
+	// ReleaseOnCancel defines if the leader should step down voluntarily
+	// when the Manager ends. This requires the binary to immediately end when the
+	// Manager is stopped, otherwise this setting is unsafe. Setting this significantly
+	// speeds up voluntary leader transitions as the new leader doesn't have to wait
+	// LeaseDuration time first.
+	ReleaseOnCancel bool
+
+	// LeaseDuration is the duration that non-leader candidates will
+	// wait to force acquire leadership. This is measured against time of
+	// last observed ack. Default is 15 seconds.
```

Comment:
Given the context of the doc block is this package, I think it should be:

```suggestion
	// last observed ack. Default is 30 seconds.
```

Same goes for the other documented defaults.